projects/storefrontlib/src/cms-structure/page/slot/page-slot.component.ts

```diff
@@ -130,7 +130,13 @@ export class PageSlotComponent implements OnInit, OnDestroy {
       this.class = cls;
     }
 
-    this.addSmartEditSlotClass(slot);
+    if (slot) {
+      this.dynamicAttributeService.addAttributesToSlot(
```

Comment:
Similar to components, we should fallback to `addDynamicAttributes` if there is no `addAttributesToSlot` method